### ISSUE TITLE
Validation rules should not be case sensitive except where specified in the definition

### ISSUE THREAD

**Issue by SLornieCYC** [data-to-insight/csc-validator-be-903#332]
For example:

- [ ] 149 (https://github.com/SocialFinanceDigitalLabs/quality-lac-data-beta-validator/commit/d2bbfae6cd849032338ce5861e7b6563e1077361) - "E4a" (mixed case as recorded locally) fails the validation but is allowed by the DFE return portal.

- [ ] 392c (https://github.com/SocialFinanceDigitalLabs/quality-lac-data-beta-validator/commit/cfdfba686abbf11b8e761af68470c6f956934a6d) - lower case postcode fails the validation. I can't evidence this being allowed by the DFE return portal but see no reason why it shouldn't be accepted.

**Comment by dezog** [comment]
Hey Stephen, this is a great point - well spotted!

I agree, can't think of a good reason to require upper case.

Modifying those two rules shouldn't be hard, but will have to have a think about the best way to guard against this in future rules.

**Comment by PhilDeakin** [comment]
Would it not be simpler just to code all the string fields from the CSV files into upper case before the validation rules are run against them? Then we wouldn't have to worry about such things.

**Comment by dezog** [comment]
Yeah I think you're right.  I had discussed this with a colleague a while back and we worried that if we did it that way it might end up causing problems down the line, say if the gov added some case-sensitive rules. 

However, given that that's extremely hypothetical, and adding it to every rule along with changes to the tests would be a bit of a headache at this stage, just uppercasing all the text when we load it in makes the most sense.

This will definitely have to get done by launch, since it's already caused problems. In fact it should ideally be fixed ahead of the start of user-testing on Friday, as it could trip us up at the first hurdle... 

Should be a fairly easy fix in ingress.py - happy for anyone to assign it to themselves and give it a go if interested - otherwise I'll try and get to it by tomorrow morning
